morl_baselines/single_policy/esr/eupg.py

```diff
@@ -192,6 +198,15 @@ def update(self):
                 },
             )
 
+    def forward_cumulative_rewards(self, rewards):
```

Comment:
The CI is probably going to yell because of missing docstring here. I propose to make this method private -> `_forward_cumulative_rewards`